nntrainer/tensor/blas_neon.cpp

```diff
@@ -2057,5 +2079,80 @@ void squared_root_transform_inplace_neon(const unsigned int N, __fp16 *X) {
   }
 }
 
+void compute_rotary_embedding_value_neon(unsigned int dim, unsigned int half_,
+                                         unsigned int w, __fp16 *in,
```

Comment:
It is really great to have this fuc with SIMD! Great work! One thing I would recommend is how about moving this function to util directory for management consistency.

Reply:
got it